### PR TITLE
📊  Adding diarrhea treatment dataset

### PR DESCRIPTION
**Pull request by spoonerf**

### CODE REVIEW

File: snapshots/unicef/2023-06-16/diarrhea.xlsx.dvc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+meta:
+  namespace: unicef
+  short_name: diarrhea
+  name: Diarrhea treatment (UNICEF, 2022)
+  version: '2023-06-16'
+  publication_year: 2022
+
+  publication_date: '2022-05-01'
+  source_name: UNICEF (2022)
+  source_published_by: Several health and multiple indicator cluster surveys compiled
```
Comment (paarriagadap) [inline snapshots/unicef/2023-06-16/diarrhea.xlsx.dvc:10]: Shouldn't this be a longer reference to the source? Like UNICEF (2022). Diarrhoea data or something?

File: etl/steps/data/garden/unicef/2023-06-16/diarrhea.meta.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  sources:
+  - name: UNICEF (2022)
+    url: https://data.unicef.org/resources/dataset/diarrhoea/
+    source_data_url: https://data.unicef.org/wp-content/uploads/2018/06/Child-Health-Coverage-Database-May-2022.xlsx
+    date_accessed: '2023-06-16'
+    publication_date: '2022-05-01'
+    publication_year: 2022
+    published_by: Several health and multiple indicator cluster surveys compiled by
+      UNICEF
+tables:
```
Comment (paarriagadap) [inline etl/steps/data/garden/unicef/2023-06-16/diarrhea.meta.yml:16]: If the metadata above is exactly the same as meadow you can delete it and you avoid to update it multiple times in the future

File: snapshots/unicef/2023-06-16/diarrhea.xlsx.dvc (hunk 1 above)
Comment (spoonerf) [reply snapshots/unicef/2023-06-16/diarrhea.xlsx.dvc:10]: Ah I thought that is the `name` component higher up? And then `source_name` is what should show in the bottom left of the chart?

File: etl/steps/data/garden/unicef/2023-06-16/diarrhea.meta.yml (hunk 2 above)
Comment (spoonerf) [reply etl/steps/data/garden/unicef/2023-06-16/diarrhea.meta.yml:16]: Ah thanks!

File: snapshots/unicef/2023-06-16/diarrhea.xlsx.dvc (hunk 1 above)
Comment (paarriagadap) [reply snapshots/unicef/2023-06-16/diarrhea.xlsx.dvc:10]: Isn't `name` the name of the dataset on Grapher admin?

Comment (spoonerf) [reply snapshots/unicef/2023-06-16/diarrhea.xlsx.dvc:10]: Ah yes you're right - do you mean `source_published_by` should be: UNICEF (2022). Diarrhoea data, and I should change what I currently have to `source_publisher_source` - I think?

Comment (spoonerf) [reply snapshots/unicef/2023-06-16/diarrhea.xlsx.dvc]: Oh interesting - I added it in the dvc file and it seemed to work. Let's see what happens ;)

**Pull request merged**